### PR TITLE
Basic Project Setup.

### PR DESCRIPTION
**Pull request by oke-aditya**
Helps #11 

I'm rearrangning and fixing the docs as well as added how to make PR to this repo.

I guess contributing should have how to make PR and current content in contributing is shifted to Readme so people know what this repo should do.

cc @ayulockin @GokuMohandas'

### CODE REVIEW

**Comment by oke-aditya** [comment]
This needs some addtional work. Hopefully, by tomorrow I will finish this up.

**Comment by oke-aditya** [comment]
Seems good. I guess we are almost there with the setup. I really need help in creating GitHub issues. I'm not expert and might take lot of time to create 10-15 issues (assuming 15 mins on each)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,22 +1,102 @@
...
+local machine. This is the working directory where you will make
+changes:
 
-## How can you contribute? :relaxed:
+```bash
+$ git clone https://github.com/madewithml/hacktoberfest.git
+```
 
-If you are working on an exciting ML/DL project, we would love to feature your work in this repo. The rules are simple:
+3.  Install the requirments. You many use miniconda or conda as well.
```
Comment (ayulockin) [inline CONTRIBUTING.md:20]: You *may* use

Comment (oke-aditya) [reply CONTRIBUTING.md:20]: K changed 👍

**Pull request merged**